x-pack/plugins/index_management/public/services/public_api_service.ts

```diff
@@ -0,0 +1,39 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0; you may not use this file except in compliance with the Elastic License
+ * 2.0.
+ */
+
+import { HttpSetup } from '@kbn/core/public';
+import { sendRequest } from '../shared_imports';
+import { API_BASE_PATH } from '../../common/constants';
+
+export interface PublicApiServiceSetup {
+  getAllEnrichPolicies(): void;
```

Comment:
Should we use the return type here to indicate that this method returns enrich policies?